### DISCUSSION TITLE
this-week-in-neovim/79

### DISCUSSION THREAD

**Discussion by giscus** [codicocodes/dotfyle#153] [This Week in Neovim]
# this-week-in-neovim/79

Find new plugins, trending plugins & popular plugins. Search for Neovim configurations by plugin manager, plugins or language cervers. Read the lateset Neovim News in This Week in Neovim.

https://dotfyle.com/this-week-in-neovim/79

**Comment by azhagar18** [comment]
Hi Dotfyle Team,

It's been a while since we last received a twin blog post from you, covering new plugins and Neovim updates. I just wanted to check in and see if everything is going well on your side.
